curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/6143931a113bb80c45546287.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-23
 
 # --description--
 
-Arguably, `D.O.B.` is not descriptive enough. This is especially true for visually impaired users. One way to get around such an issue, without having to add visible text to the label, is to add text only a screen reader can read.
+Although the `Date of Birth` text is descriptive, it can be problematic for visually impaired users. One way to get around such an issue is to add text only a screen reader can read.
```

Comment:
> Any ideas on how to incorporate the `sr-only` class for this lesson?

The first item under "Should Fix":

> "Question 1" and "Question 2" sr-only text be added to the beginning of their respective questions so that screen reader users navigating by tab stop will hear the question number as well as the question text.

_Originally posted by @bbsmooth in https://github.com/freeCodeCamp/freeCodeCamp/issues/47830#issuecomment-2116694307_

Will need to create another PR to address `sr-only` text for "Question 1" and "Question 2" headings, as separate PRs are required for the issue.